lib/traject/mods_config.rb

```diff
@@ -2,24 +2,159 @@
+to_field 'cho_has_type', extract_mods('/*/mods:genre')
+to_field 'cho_is_part_of', generate_part_of
+to_field 'cho_is_part_of', generate_series
+to_field 'cho_publisher', extract_mods('/*/mods:originInfo/mods:publisher')
+to_field 'cho_relation', generate_relation
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:cartographics/mods:coordinates')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:cartographics/mods:projection')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:cartographics/mods:scale')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:geographic')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:geographicCode')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:area')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:city')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:citySection')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:continent')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:country')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:county')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:extraterrestrialArea')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:island')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:region')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:state')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:territory')
+to_field 'cho_subject', extract_mods('/*/mods:classification')
+to_field 'cho_subject', extract_mods('/*/mods:subject/mods:topic')
+to_field 'cho_subject', extract_mods('/*/mods:subject/mods:titleInfo/mods:title')
+to_field 'cho_temporal', extract_mods('/*/mods:subject/mods:temporal')
+to_field 'cho_type', extract_mods('/*/mods:typeOfResource')
+
+# Aggregation Object(s)
+# flat fields
+to_field 'agg_data_provider', lambda { |_record, accumulator, context|
+  accumulator << context.settings.fetch('agg_data_provider')
+}
+to_field 'agg_provider', lambda { |_record, accumulator, context|
+  accumulator << context.settings.fetch('agg_provider')
+}
+# agg_dc_rights:,
+# agg_edm_rights:,
+# agg_same_as
+
+# nested fields
+
+# Not using agg_has_view since we have the above
+to_field 'agg_is_shown_at' do |record, accumulator, context|
+  druid = generate_druid(record, context)
+
+  accumulator << transform_values(context, wr_id: literal(generate_sul_shown_at(record, druid)))
+end
+
+to_field 'agg_is_shown_by' do |record, accumulator, context|
+  druid = generate_druid(record, context)
+  manifest = "https://purl.stanford.edu/#{druid}/iiif/manifest"
+  iiif_json = grab_sul_iiif_links(manifest)
```

Comment:
sorry, just seeing this comment. i was hoping to handle the error in the IIIF methods (albeit they still have ParserError from a recent rewrite, which you clocked already as incorrect). We'll probably end up needing both checks.